src/components/Wrapper.js

```diff
@@ -16,15 +15,14 @@ import {
 } from 'ramda';
 import { DesktopOnly, MobileOnly } from './MediaQuery';
 import MenuButtons from './MenuButtons';
-import Spacer from './Spacer';
 
 const drawerWidth = 142;
 const headerHeight = 50;
 
 const Header = styled.div`
   width: 100%;
   height: ${headerHeight}px;
-  background-color: gray;
+  background-color:#0079D3 ;
```

Comment:
Could you go ahead and create a `colors.js` module that just exports an object something like this:
```
export default {
    white: '#fff',
    blue: '#0079D3',
    ....
};
```
We can just keep our app color palette in there so our colors can stay consistent and it makes it easier to change later.

Reply:
Yeah, I'll work on this and throw it into the utils folder

Reply:
Done